integration/integration_test.go

```diff
@@ -5094,9 +5101,13 @@ func (t *Terminal) Write(data []byte) (n int, err error) {
 	return t.written.Write(data)
 }
 
+func (t *Terminal) Close() {
```

Comment:
nit: rename to something like `closeSend` to make it clearer that this doesn't close the entire terminal